Makefile

```diff
@@ -70,17 +70,13 @@ endif
 
 # control rln code compilation
 ifeq ($(RLN), true)
-NIM_PARAMS := $(NIM_PARAMS) -d:rln
-else  ifeq ($(CI), true)
-NIM_PARAMS := $(NIM_PARAMS) -d:rln 
+NIM_PARAMS := $(NIM_PARAMS) -d:rlnzerokit
+else ifeq ($(CI), true)
+NIM_PARAMS := $(NIM_PARAMS) -d:rlnzerokit 
```

Comment:
Can you please make another rename from `rln-> rlnkilic` and then `rlnzerokit` -> `rln`, just want to make sure we wont need to change anything on the test fleets.

Reply:
You may skip my suggested change for now. I just checked the docker file and it seems that it only depends on the `RLN` variable to pass the compiler flag `-d:rln`, then if that assumption is true, I expect that by just swapping the behavior of `rln` to compile zerokit instead of killic everything else should be fine.
Lets merge this one and see what is the outcome.